cmd/util/misc.go

```diff
@@ -94,3 +94,26 @@ func GetKVSecret(kvName, kvSecretName string) (secret string, err error) {
 
 	return *secretResp.Value, nil
 }
+
+func GetBlockSize(filesize int64, minBlockSize int64) (blockSize int64) {
+	blockSizeThreshold := int64(256 * 1024 * 1024) /* 256 MB */
+	blockSize = minBlockSize
+
+	/* We should not perform checks on filesize, block size limitation here. Those are performed in SDK
```

Comment:
We should consider whether to fail early and fast if the file is too large, perhaps add a TODO here. Because the files in Lustre can get really large and it'd be great to not waste time on uploading if it's going to fail after a while.

Reply:
The SDK also fails early, even before we upload/download anything.